lib/tasks/report_inconsistent_facet_values.rake

```diff
@@ -0,0 +1,57 @@
+require "gds_api/rummager"
+require "json"
+
+class EmptyFacetValueError < StandardError
+end
+
+desc "Check for and report any non-expanded facet values"
+task :report_inconsistent_facet_values do
+  # List of facets to check
+  facets = %w{
```

Comment:
I suspect you could use `Rummager.registries. as_hash.keys.map(&:to_s)`. If not then copy paste is fine!